ddtrace/appsec/processor.py

```diff
@@ -313,10 +381,18 @@ def on_span_finish(self, span):
 
             if _Addresses.SERVER_RESPONSE_HEADERS_NO_COOKIES in data:
                 _set_headers(span, data[_Addresses.SERVER_RESPONSE_HEADERS_NO_COOKIES], kind="response")
+
+            if not blocked_request and ddwaf_result:
+                # Partial DDAS-011-00
+                log.debug("[DDAS-011-00] AppSec In-App WAF returned: %s", ddwaf_result.data)
+                span.set_tag_str(APPSEC_JSON, '{"triggers":%s}' % (ddwaf_result.data,))
+            else:
+                span.set_tag(APPSEC_JSON, _context.get_item("http.request.waf_json", span=span))
+                span.set_tag(APPSEC_JSON, _context.get_item("http.request.waf_json", span=span))
+                span.set_tag("appsec.blocked", True)
```

Comment:
```suggestion
                span.set_tag_str("appsec.blocked", "true")
```

assuming this is what you want, since below you also have `set_tag_str("appsec.event", "true")` ?

Reply:
Yup.